kyaml/runfn/runfn.go

```diff
@@ -25,12 +29,18 @@ type RunFns struct {
 	// directory
 	FunctionPaths []string
```

Comment:
Might want to update the comment on this field since `functions provided on FunctionPaths are globally scoped`. Also maybe mention that these functions run after implicitly found functions.

Reply:
Done